torch/testing/_internal/common_methods_invocations.py

```diff
@@ -11249,6 +11291,37 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                DecorateInfo(toleranceOverride({torch.float16: tol(atol=1e-02, rtol=1e-02)}),
                             'TestCudaFuserOpInfo', 'test_nvfuser_correctness'),
            )),
+    OpInfo('nn.functional.linear',
```

Comment:
After various offline discussion I think this is the best solution for testing ATM. 

The new OpInfo variant generates sparse samples exclusively and adds xfails with are problematic due to the test not yet handling sparsity naturally.